### PR TITLE
docs: footer updated and post deployment guide name change

### PR DESCRIPTION
**Pull request by shyogesh-sw**
Issue #, if available: https://sim.amazon.com/issues/GALI-1036

Description of changes: Changed the footer and name of "post deployment" section to Service Workbench Adminstrator's Guide".

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (SanketD92) [inline yarn.lock]: Please delete this file. We don't use yarn on the root directory.

File: docs/sidebars.js [hunk 2]
```diff
@@ -42,39 +42,22 @@ module.exports = {
...
-          "deployment/post_deployment/create_index_project",
-          "deployment/post_deployment/create_admin_user",
-          "deployment/post_deployment/import_service_catalog_products",
-          "deployment/post_deployment/logs",
-        ],
-      },
+    "Service Workbench Administrator's Guide": [
+      "deployment/post_deployment/index",
+      "deployment/post_deployment/account_structure",
+      // "deployment/post_deployment/cost_explorer",
```
Comment (SanketD92) [inline docs/sidebars.js]: Could we remove this commented code?

File: yarn.lock (hunk 1 above)
Comment (shyogesh-sw) [reply yarn.lock]: Deleted the file.

Comment (SanketD92) [reply yarn.lock]: Looks like the file is still there.

**Review by SanketD92 — APPROVED** [review]

**Pull request closed without merging**